pants.toml

```diff
@@ -134,7 +134,7 @@ extra_requirements.add = [
   "flake8-no-implicit-concat",
   "flake8-comprehensions>=3.8.0,<4.0",
 ]
-lockfile = "3rdparty/python/lockfiles/flake8.txt"
+lockfile = "3rdparty/python/flake8.lockfile"
```

Comment:
Style nit. Why naming the lockfiles `*.lockfile` rather than just `*.lock`? It clearly is a file, so seems a bit redundant noise to me. (`*.txt` vs `*.txtfile` etc..)

```python
lockfile = "3rdparty/python/flake8.lock"
```

Reply:
Yeah I agree with that tbh...I'm not sure why we did `.lockfile` with JVM. @stuhood do you remember?

Reply:
No, I do not.